test/suites/clustering.sh

```diff
@@ -4401,3 +4401,81 @@ test_clustering_trust_add() {
   kill_lxd "${LXD_ONE_DIR}"
   kill_lxd "${LXD_TWO_DIR}"
 }
+
+test_clustering_link() {
+  local LXD_DIR
+
+  LXD_ONE_DIR=$(mktemp -d -p "${TEST_DIR}" XXX)
+  chmod +x "${LXD_ONE_DIR}"
+  spawn_lxd "${LXD_ONE_DIR}" false
+
+  LXD_TWO_DIR=$(mktemp -d -p "${TEST_DIR}" XXX)
+  chmod +x "${LXD_TWO_DIR}"
+  spawn_lxd "${LXD_TWO_DIR}" false
+
+  # shellcheck disable=SC2034,SC2030
+  LXD_DIR=${LXD_ONE_DIR}
+
+  # Get the address of LXD_ONE.
+  LXD_ONE_ADDR="$(lxc config get core.https_address)"
+
+  # Enable clustering on LXD_ONE.
+  lxc cluster enable node1
+  [ "$(lxc cluster list | grep -cF 'node1')" = 1 ]
+
+  # Get a cluster link trust token from LXD_ONE.
+  lxc cluster link add lxd_two --quiet > "${TEST_DIR}/lxd_one_trust_token"
+
+  # The cluster link identity on LXD_ONE should be pending.
+  [ "$(lxc auth identity list --format csv | grep -cF 'Cluster link certificate (pending)')" = 1 ]
+
+  LXD_DIR=${LXD_TWO_DIR}
+
+  # Get the address of LXD_TWO.
+  LXD_TWO_ADDR="$(lxc config get core.https_address)"
+
+  # Enable clustering on LXD_TWO.
+  lxc cluster enable node1
+  [ "$(lxc cluster list | grep -cF 'node1')" = 1 ]
```

Comment:
```suggestion
  [ "$(lxc cluster list | grep -cwF 'node1')" = 1 ]
```

By forcing a match on full word you alleviate the risk of matching on a theoretical `node11`